webpack/scenes/ContentViews/Create/CreateContentViewForm.js

```diff
@@ -0,0 +1,94 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory, Redirect } from 'react-router-dom';
+import {Modal, Form, FormGroup, Popover, TextInput, Input,
+  Checkbox,
+  ActionGroup, ModalVariant, Button} from '@patternfly/react-core';
+import { createContentView } from '../ContentViewsActions';
+
+const ContentViewForm = ({ setModalOpen }) => {
+  const dispatch = useDispatch();
+  let history = useHistory();
+  const [created, setCreated] = useState(false);
+  const [name, setName] = useState('');
+  const [label, setLabel] = useState('');
+  const [description, setDescription] = useState('');
+  const [composite, setComposite] = useState(false);
+  const toggleComposite = () => {
+    setComposite(!composite);
+  };
+  const [dependencies, setDependencies] = useState(false);
+  const toggleDependencies = () => {
+    setDependencies(!dependencies);
+  };
```

Comment:
are these wrapper functions going away? I don't see them in use anywhere

Reply:
Yes..these aren't needed..